sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtilsTest.java

```diff
@@ -452,6 +457,27 @@ public void testMicroPrecisionTimeType() {
         equalTo(t));
   }
 
+  @Test
+  public void testMicroPrecisionDateTimeType() {
+    LocalDateTime dt = LocalDateTime.parse("2020-06-04T12:34:56.789876");
+    assertThat(
+        BigQueryUtils.convertAvroFormat(
+            FieldType.logicalType(SqlTypes.DATETIME), new Utf8(dt.toString()), REJECT_OPTIONS),
+        equalTo(dt));
+  }
+
+  @Test
+  public void testNumericType() {
+    // BigQuery NUMERIC type has precision 38 and scale 9
+    BigDecimal n = new BigDecimal("123456789.987654321").setScale(9);
+    assertThat(
+        BigQueryUtils.convertAvroFormat(
+            FieldType.DECIMAL,
+            new Conversions.DecimalConversion().toBytes(n, null, LogicalTypes.decimal(38, 9)),
+            REJECT_OPTIONS),
+        equalTo(n));
+  }
+
```

Comment:
Could you make sure that `BigQueryUtils.fromTableSchema` is tested with NUMERIC and DATETIME? It looks like only TIMESTAMP may be tested now (as part of `testTableFromSchema_flat`): https://github.com/apache/beam/blob/738a9100ec5ce09870729b64e1ef483f692134c2/sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtilsTest.java#L91-L106

Reply:
Done. Added test for new types: DATE, TIME, and DATETIME